### ISSUE TITLE
npm WARN deprecated puppeteer@21.0.1: < 21.5.0 is no longer supported

### ISSUE THREAD

**Issue by osti-barrozo** [frinyvonnick/node-html-to-image#224]
How to solve depecation?

`npm WARN deprecated puppeteer@21.0.1: < 21.5.0 is no longer supported`

**Comment by maheshjq** [comment]
Yes i'm having the same concern. Thanks.

**Comment by shincha3517** [comment]
same issue

**Comment by Davenporten** [comment]
Yeah, I'm wondering if this lib is still supported. @frinyvonnick not a criticism, just curious, the last release is coming up on a year.

**Comment by frinyvonnick** [comment]
Sure, it is still supported 😄 Everyone is welcome to open pull requests to fixes issues. Otherwise, I'll take care of it as soon as my personal life allows it 😉

**Comment by Davenporten** [comment]
Totally get it, just wanted to check. Thanks for the great lib!

**Comment by sriram15690** [comment]
Yes I'm having the same concern as well.

**Comment by frinyvonnick** [comment]
I updated puppeteer to 23.2.2. You can update node-html-to-image in v5.0.0 this might fix your issue 😄 Can you please close this issue if it is the case?

**Comment by maheshjq** [comment]
Thanks @frinyvonnick , let me check this in my projects.

**Comment by frinyvonnick** [comment]
This was tied to an old Puppeteer version. The latest release (v6) ships Puppeteer 25 with the new headless mode, so this no longer applies. Please upgrade to the latest version. Closing — feel free to reopen if it still happens on v6.
